tests/docker/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG jdk_version=openjdk:11
+ARG jdk_version=openjdk:17.0.2-buster
```

Comment:
The base image of the  openjdk:17 are almost `oraclelinux:8-slim`, this image doesn't have `apt-get`, thus I need to use `openjdk:17.0.2-buster`

Reply:
please add comment to the dockerfile

Reply:
Additionally, could you use `openjdk:17-buster` instead? that can point to the latest release